pkg/ocm/policy.go

```diff
@@ -26,10 +26,47 @@ type PolicyBuilder struct {
 	errorMsg string
 }
 
+// NewPolicyBuilder creates a new instance of PolicyBuilder.
+func NewPolicyBuilder(apiClient *clients.Settings, name, nsname string) *PolicyBuilder {
```

Comment:
Can you create policy with name and namespaces only? Does this resource have any other required parameter?

Reply:
I tried to create a policy and it does look like having at least one policy template is required, so I'll go ahead and add that parameter